src/client/terrain/util/mesher.rs

```diff
@@ -75,7 +78,7 @@ pub fn create_cube_geometry_data(
     }
 }
 
-pub fn create_chunk_mesh(chunk: &Chunk) -> Option<Mesh> {
+pub fn create_chunk_mesh(chunk: &Chunk, texture_manager: &TextureManager) -> Option<Mesh> {
```

Comment:
Introducing TextureManager resource does increase complexity but at least that shit doesn't need to be hardcoded anymore (the constructor could dynamically set texture coordinates)